doc/getting-started.md

```diff
@@ -1,7 +1,8 @@
-# CHES CHERIoT Workshop Preparation
+# Getting started guide
 
-In advance of the CHES Affiliated Sunburst Workshop, it's essential you prepare your environment so any issues can be sorted in advance of the day.
-If you have any issues in following this guide please contact the Sunburst Team on [ches-2024@lowrisc.org](mailto:ches-2024@lowrisc.org).
+This guide tells you how to prepare your environment before working with Sonata.
+The guide is especially useful to follow in advance of organized workshops or events so any issues can be sorted in advance of the day.
+If you have any issues in following this guide please contact the Sunburst Team on [info@lowrisc.org](mailto:info@lowrisc.org).
```

Comment:
```suggestion
If you have any issues in following this guide please contact the Sunburst Team at [info@lowrisc.org](mailto:info@lowrisc.org).
```

I know this wasn't due to your changes, but only just spotted it.

Reply:
Thanks added that in.